### DISCUSSION TITLE
2 entities not available since 13.1.1 update

### DISCUSSION THREAD

**Discussion by billmars206** [BottlecapDave/HomeAssistant-OctopusEnergy#1074] [Q&A] [answered]
Since update 13.1.1, I'm getting:-

This entity is no longer being provided by the octopus_energy integration.

number.octopus_energy_a_xxxxxxxx_intelligent_charge_limit

time.octopus_energy_a_xxxxxxxx_intelligent_ready_time

Is it me or is there a reason?

Thanks

**Comment by BottlecapDave** [comment, marked as answer]
Hello. I'm guessing you've jumped a few versions up to 13.1.1? As part of v13.0.0, these entities were renamed to charge_target and target_time to be inline with the underlying API and be more inclusive of different uses of these sensors depending on which provider you have for intelligent. This was highlighted as a breaking change in the release notes.

**Reply by billmars206** [reply]
Many thanks.
Must have missed reading the release notes.
Mea culpa!
